canonical/snapcraft.io
# Fix snap builds page not working

## Done
- Fix snap builds page not working when snap base is `core`
## How to QA
- If you have snap whose base is `core` (without any number attached), Go to https://snapcraft-io-4188.demos.haus/<snap-name>/builds. Ensure that the snaps build page displays properly
## Issue / Card WD-1252
Fixes #4185

## Screenshots
